### DISCUSSION TITLE
[Question][Hw4] Editing json_seed

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#571] [Q&A]
In the instruction, the function update_seeds is explained as "adds a given input into the seed pool if it is interesting.". Can I remove some elements from the seed corpus also? Or can I just keep adding elements, and can't remove added elements?

**Comment by gunsodo** [comment]
I think this function is for updating the seeds only and you can do whatever you want with the function as long as it intuitively works as declared. Also, I think it should be used in `run` as well.

**Comment by ee12ha0220** [comment]
Thanks:)

**Comment by RiceBiscuits** [comment]
If removing the seed from the updated_seeds can make performance of your fuzzer better, It is up to you.

**Comment by ee12ha0220** [comment]
Oh thanks!
